### DISCUSSION TITLE
Adding the collected Explorernotes to the Player Sites

### DISCUSSION THREAD

**Discussion by InfinityofphoenixTimo** [gameserverapp/Platform#883] [Suggestions]
Add collected Explorer Notes to the player profile page. I've seen sites that do this, it must be possible to extract them. Ideally with pictures of the maps, and for the Explorer Notes + Jane Notes etc. locations and a picture. The locations and pictures could be collected by the community.

Perhaps this could be something that could be attached to the suggestion of completed boss fights etc.

**Comment by mvaessen** [comment]
Since this is very specific to ark, is it not something I see any benefit.
